.github/release-drafter.yml

```diff
@@ -14,10 +14,10 @@ categories:
     label: removed
   - title: ⚠️ Deprecated
     label: deprecated
-  - title: 🚀 New features and improvements
-    labels:
-      - enhancement
-      - new-feature
+  - title: ":tada: New features"
```

Comment:
#470?

Reply:
Right. Maybe just sticking with a single label is better